www/src/components/docs-table-of-contents.js

```diff
@@ -0,0 +1,61 @@
+import React from "react"
+import { Link } from "gatsby"
+import {
+  fontSizes,
+  colors,
+  zIndices,
+  shadows,
+  space,
+  mediaQueries,
+} from "../utils/presets"
+
+function createItems(items, location) {
+  return (
+    items &&
+    items.map(item => (
+      <li>
+        <Link to={location.pathname + item.url}>{item.title}</Link>
+        {item.items && (
+          <ul css={{ marginLeft: space[6] }}>
+            {createItems(item.items, location)}
+          </ul>
+        )}
+      </li>
+    ))
+  )
+}
+
+function TableOfContents({ page, location }) {
+  return page.tableOfContents.items ? (
+    <div
+      css={{
+        background: colors.white,
+        zIndex: zIndices.feedbackWidget,
+
+        [mediaQueries.xxl]: {
+          padding: space[6],
+          boxShadow: shadows.overlay,
```

Comment:
IMO the shadow isn't totally necessary, I think it looks good sitting flat and it feels a little more visually consistent (with the rest of the page and with the mobile view as well) to me without it:

![image](https://user-images.githubusercontent.com/21114044/60445051-afbc3100-9bdb-11e9-88b0-610e0001867d.png)

```suggestion
```

Reply:
👍